bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java

```diff
@@ -209,11 +209,9 @@ public void updateTransactions(Account account, Urllib urlopen) throws LoginExce
 			
 			account.setTransactions(transactions);
 		} catch (ClientProtocolException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
+			throw new BankException(e.getMessage(), e);
 		} catch (IOException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
-		}
+            throw new BankException(e.getMessage(), e);
+        }
```

Comment:
I will verify this. However, the formatting of all files should be done in a separate PR